Common/itkComputePreconditionerUsingDisplacementDistribution.h

```diff
@@ -0,0 +1,160 @@
+/*=========================================================================
+ *
+ *  Copyright UMC Utrecht and contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef __itkComputePreconditionerUsingDisplacementDistribution_h
+#define __itkComputePreconditionerUsingDisplacementDistribution_h
+
+#include "itkComputeDisplacementDistribution.h"
+
+
+namespace itk
+{
+/**\class ComputePreconditionerUsingDisplacementDistribution
+ * \brief This is a helper class for the automatic estimation of a preconditioner for the FPSGD optimizer.
+ * // update below
+ * More specifically this class computes the Jacobian terms related to the automatic
+ * parameter estimation for the adaptive stochastic gradient descent optimizer.
+ * Details can be found in the SPIE paper
+ *
+ * [1]  Y.Qiao, B.P.F. Lelieveldt, M.Staring
```

Comment:
Qiao Y, Lelieveldt BP, Staring M. An efficient preconditioner for stochastic gradient descent optimization of image registration. IEEE transactions on medical imaging. 2019 Feb 11.